README.md

```diff
@@ -52,8 +52,8 @@ Remember to the add a dependency entry to your `fabric.mod.json` file:
             }
         ]
     },
-    "depends": {
-        "fabric-language-kotlin": ">=1.3.72"
+    "requires": {
```

Comment:
This should still be `depends`. `requires` a [deprecated leftover](https://github.com/FabricMC/fabric-loader/blob/031235cfa3b028a75ab6d72ff5701648ff9fae5f/src/main/java/net/fabricmc/loader/metadata/ModMetadataV1.java#L116) from loader version before 0.4.